regulation-worker/internal/delete/delete_test.go

```diff
@@ -0,0 +1,180 @@
+// 			job: model.Job{
+// 				ID:            1,
+// 				WorkspaceID:   "1001",
+// 				DestinationID: "1234",
+// 				Status:        model.JobStatusPending,
+// 				UserAttributes: []model.UserAttribute{
+// 					{
+// 						UserID: "12",
+// 						Phone:  strPtr("1234567890"),
+// 						Email:  strPtr("abc@xyz.com"),
+// 					},
+// 					{
+// 						UserID: "13",
+// 						Email:  strPtr("abcd@xyz.com"),
+// 					},
+// 					{
+// 						UserID: "14",
+// 						Phone:  strPtr("1111567890"),
+// 					},
+// 				},
+// 			},
+// 			dest: model.Destination{
+// 				Config: map[string]interface{}{
+// 					"bucketName":  "malani-deletefeature-testdata",
+// 					"prefix":      "regulation",
+// 					"accessKeyID": "abc",
+// 					"accessKey":   "xyz",
+// 					"enableSSE":   false,
+// 				},
+// 				DestinationID: "1234",
+// 				Type:          "api",
+// 				Name:          "mock_api",
+// 			},
+// 		},
+// 		{
+// 			name: "testing KVStore deletion flow by deletion from 'mock_kvstore' destination",
+// 			job: model.Job{
+// 				ID:            1,
+// 				WorkspaceID:   "1001",
+// 				DestinationID: "1234",
+// 				Status:        model.JobStatusPending,
+// 				UserAttributes: []model.UserAttribute{
+// 					{
+// 						UserID: "12",
+// 						Phone:  strPtr("1234567890"),
+// 						Email:  strPtr("abc@xyz.com"),
+// 					},
+// 					{
+// 						UserID: "13",
+// 						Email:  strPtr("abcd@xyz.com"),
+// 					},
+// 					{
+// 						UserID: "14",
+// 						Phone:  strPtr("1111567890"),
+// 					},
+// 				},
+// 			},
+// 			dest: model.Destination{
+// 				Config: map[string]interface{}{
+// 					"bucketName":  "malani-deletefeature-testdata",
+// 					"prefix":      "regulation",
+// 					"accessKeyID": "abc",
+// 					"accessKey":   "xyz",
+// 					"enableSSE":   false,
+// 				},
+// 				DestinationID: "1234",
+// 				Type:          "kv_store",
+// 				Name:          "mock_kvstore",
+// 			},
+// 		},
+// 		{
+// 			name: "testing KVStore deletion flow by deletion from 'mock_kvstore' destination",
+// 			job: model.Job{
+// 				ID:            1,
+// 				WorkspaceID:   "1001",
+// 				DestinationID: "1234",
+// 				Status:        model.JobStatusPending,
+// 				UserAttributes: []model.UserAttribute{
+// 					{
+// 						UserID: "12",
+// 						Phone:  strPtr("1234567890"),
+// 						Email:  strPtr("abc@xyz.com"),
+// 					},
+// 					{
+// 						UserID: "13",
+// 						Email:  strPtr("abcd@xyz.com"),
+// 					},
+// 					{
+// 						UserID: "14",
+// 						Phone:  strPtr("1111567890"),
+// 					},
+// 				},
+// 			},
+// 			dest: model.Destination{
+// 				Config: map[string]interface{}{
+// 					"bucketName":  "malani-deletefeature-testdata",
+// 					"prefix":      "regulation",
+// 					"accessKeyID": "abc",
+// 					"accessKey":   "xyz",
+// 					"enableSSE":   false,
+// 				},
+// 				DestinationID: "1234",
+// 				Type:          "random",
+// 				Name:          "mock_random",
+// 			},
+// 		},
+// 	}
+// 	for _, tt := range tests {
```

Comment:
what if we uncomment but skip them
```suggestion
 	for _, tt := range tests {
        t.Skip()
```